packages/truffle/scripts/test.sh

```diff
@@ -7,9 +7,9 @@ if [ "$GETH" == true ]; then
 elif [ "$FABRICEVM" == true ]; then
   mocha --timeout 50000 --grep @fabric-evm --colors $@
 elif [ "$COVERAGE" == true ]; then
-  NO_BUILD=true mocha --no-warnings --timeout 7000 --grep @geth --invert --colors $@
+  NO_BUILD=true mocha --no-warnings --timeout 20000 --grep @geth --invert --colors $@
 elif [ "$INTEGRATION" == true ]; then
-  mocha --no-warnings --timeout 7000 --grep @geth --invert --colors $@
+  mocha --no-warnings --timeout 20000 --grep @geth --invert --colors $@
 else
-  yarn build && mocha --no-warnings --timeout 7000 --grep @geth --invert --colors $@
+  yarn build && mocha --no-warnings --timeout 20000 --grep @geth --invert --colors $@
```

Comment:
these are unfortunate :/ promises slowwww. we should do further testing... we really can't afford to make truffle slower.

Reply:
We will need to try and test the speed some other way. Raising the timeouts was not necessarily because they were slow. I got rid of a lot of the timeouts specified in the individual tests as I think a lot of them were not working.